Documentation/contributing/development/images.rst

```diff
@@ -198,91 +198,52 @@ Image dependency:
 Update cilium-builder and cilium-runtime images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Login to quay.io with your credentials to the repository that you want to
-update:
+1. cilium-builder depends on cilium-runtime so one needs to update
+   cilium-runtime first. Steps 4 and 7 will fetch the digest of the image built
+   by GitHub actions.
 
-* `cilium-builder <https://quay.io/repository/cilium/cilium-builder?tab=builds>`__ - contains Cilium build-time dependencies
-* `cilium-runtime <https://quay.io/repository/cilium/cilium-runtime?tab=builds>`__ - contains Cilium run-time dependencies
+.. code-block:: shell-session
 
-0. After login, select the tab "builds" on the left menu.
+    $ make -C images/ update-runtime-image
 
-.. image:: ../../images/cilium-quayio-tag-0.png
-    :align: center
+2. Commit your changes and create a PR in cilium/cilium.
 
-1. Click on the wheel.
-2. Enable the trigger for that build trigger.
+.. code-block:: shell-session
 
-.. image:: ../../images/cilium-quayio-tag-1.png
-    :align: center
+    $ git commit -s images/ -m "update cilium-{runtime,builder}"
 
-3. Confirm that you want to enable the trigger.
+3. Ping one of the members of `team/build <https://github.com/orgs/cilium/teams/build/members>`_
+   to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`_.
```

Comment:
Just a question for clarification, doesn't need to be added to the docs necessarily: As a member of that team, how will I approve the build? Currently, I don't see any option there to approve. Will the option show up once there is a PR from a contributor?

Reply:
You will see it there once a contributor opens a PR that changes any of the files in `images/builder` or in `images/runtime`